From 1dc360790d6cdaaa124b299e8d375e3670863312 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 8 Jan 2015 14:53:30 +0100
Subject: [PATCH] Makefile: add -Wformat to CFLAGS

-Werror=format-security seems to be ignored without -Wformat

Thanks @ Christoph Brill
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 7127368..41ae274 100644
--- a/Makefile
+++ b/Makefile
@@ -13,7 +13,7 @@ sysconfdir ?= ${prefix}/etc
 bindir ?= ${prefix}/bin
 mandir ?= ${prefix}/share/man
 CFLAGS ?= -O2 -g -Wall
-CFLAGS += -std=gnu99  -D_FILE_OFFSET_BITS=64 -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2
+CFLAGS += -std=gnu99 -D_FILE_OFFSET_BITS=64 -Wformat -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2
 bashcompletiondir ?= ${datadir}/bash-completion/completions
 
 man1pages = lsinitrd.1
